src/services/sendEmailService.ts

```diff
@@ -0,0 +1,55 @@
+const sendGridMail = require('@sendgrid/mail');
+
+import sendEmail11 from "src/controllers/sendEmailController";
+
+ var sendEmail1 = async (shifts,emailToAgent) => {
+
+sendGridMail.setApiKey('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');
```

Comment:
please move apikey to environment variables

Reply:
moved, not yet tested (there were some issues before)